libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -197,6 +197,26 @@ bool AP_AHRS::airspeed_estimate(float *airspeed_ret) const
     return false;
 }
 
+// return filtered airspeed estimate if available
+bool AP_AHRS::EAS_estimate(float *airspeed_ret) const
+{
+    if (airspeed_sensor_enabled()) {
+        *airspeed_ret = _airspeed->get_EAS();
+        if (_wind_max > 0 && AP::gps().status() >= AP_GPS::GPS_OK_FIX_2D) {
+            // constrain the airspeed by the ground speed
+            // and AHRS_WIND_MAX
+            const float gnd_speed = AP::gps().ground_speed();
+            float true_airspeed = *airspeed_ret * get_EAS2TAS();
```

Comment:
You're effectively applying the EAS2TAS correction twice, because you've already asked for the EAS value.

Reply:
get_EAS is not returning TAS

Reply:
Argh, yup you are pulling it back out afterwards, sorry.